app/core/src/main/res/layout/message_list_widget_layout.xml

```diff
@@ -12,18 +12,14 @@
         android:gravity="center_vertical"
         android:orientation="horizontal">
 
-        <TextView
-            android:id="@+id/folder"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
+        <Button
+            android:id="@+id/enter_post"
+            style="@style/Widget.AppCompat.Button.ButtonBar.AlertDialog"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
             android:layout_weight="1"
-            android:paddingBottom="12dp"
-            android:paddingLeft="8dp"
-            android:paddingRight="8dp"
-            android:paddingTop="12dp"
-            android:textSize="20sp"
-            android:textColor="@color/message_list_widget_header_text"
-            tools:text="Unified Inbox" />
+            android:text="Inbox"
```

Comment:
The widget header text was getting properly localized before. Now it's a (different) hardcoded string.